src/arrayMethodSort.js

```diff
@@ -3,9 +3,38 @@
 /**
  * Implement method Sort
  */
+const defaultFunction = (a, b) => {
+  const aString = a.toString();
+  const bString = b.toString();
+
+  if (aString > bString) {
+    return 1;
+  } else if (aString < bString) {
+    return -1;
+  } else {
+    return 0;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultFunction) {
```

Comment:
this line would be enough for default function

```suggestion
  [].__proto__.sort2 = function(
     compareFunction = (a, b) => a.toString() > b.toString()
  ) {
```